File: Task14/addressbook/src/app/employee.ts
export class Employee{
    id: number;
    name: string;
    email: string;
    mobile: string;
    landline: string;
    website: string;
    address: string;
    constructor(id: number = null,
            name:string = '',
            email: string = '',
            mobile: string = '',
            landline: string = '',
            website: string = '',
            address: string = '')
    {
        this.id = id;
        this.name = name;
        this.email = email;
        this.mobile = mobile;
        this.landline = landline;
        this.website = website;
        this.address = address;
    }
}

File: Task25/Tests/AddressBookV2/AddressBookV2/Models/Employee.cs
using System.ComponentModel.DataAnnotations;
using Microsoft.AspNetCore.Mvc;

namespace AddressBookV2.Models
{
    public class Employee
    {
        public int ID { get; set; }

        [Required, StringLength(50, MinimumLength = 1)]
        public string Name { get; set; }

        [Required, EmailAddress]
        [Remote("CheckExistingEmail", "Employee", ErrorMessage = "E-Mail already Exists")]
        public string Email { get; set; }

        [Required, StringLength(10)]
        public string MobileNumber { get; set; }

        public string Landline { get; set; }

        [Url]
        public string Website { get; set; }

        [Required, StringLength(100)]
        public string Address { get; set; }
    }
}
